### ISSUE TITLE
Missing features in Firefox?

### ISSUE THREAD

**Issue by miketaylr** [jackschaedler/goya#1]
According to your demo, Firefox Nightly is missing some features required to play with it. Are those documented anywhere? https://github.com/jackschaedler/goya/blob/gh-pages/browser-check.js doesn't give many clues.

thanks!

**Comment by heyLu** [comment]
If you remove the check, the basic features work. There are a few things I noticed, though:
- the background of the center area (not the image itself) is white, not grey
- the color picker doesn't work (in Firefox 28, works in Nightly)
- it seems to be slower

**Comment by jackschaedler** [comment]
Yeah, as you rightly say this needs to be documented and sorted out. I put the blocker in because it looked like Chrome was the only browser currently supporting the color input type. Others have suggested just showing a banner somewhere on screen saying that only Chrome has been tested. Might look into that this weekend.

**Comment by miketaylr** [comment]
You can check https://miketaylr.com/code/html5-forms-ui-support.html for color input type support. Or http://caniuse.com/#feat=input-color for a quicker glance. But basically Chrome, Opera, and Firefox (and a handful of mobile browsers) all support this now.

**Comment by swannodette** [comment]
WebKit Nightly also supports it.

**Comment by jackschaedler** [comment]
Thanks for the help guys!

I've just merged an update which explicitly checks for color input support instead of having the stupid chrome-only check. I've tested with Firefox, Opera, and Chrome. Everything seems to work fine from my end. I did not check with WebKit, because I'm still on OSX 10.7 and I can't even run the nightly on my machine.

If the user has a browser which is not supported, they are directed to download one of the three mentioned above with relevant links.

Sorry about the original issue. I really don't want people to think badly of Clojurescript because of something silly like this.

I've also updated some of the CSS which was too Chrome-specific (i.e fixed the white background amongst some other quirks).
